### PR TITLE
Create logs screen

### PR DESCRIPTION
**Pull request by nfillion21**

### CODE REVIEW

**Comment by nfillion21** [comment]
photo_2022-05-13 15 15 58

This screen allows to view share the app logs or view it locally.

File: phoenix-android/src/main/res/drawable/ic_eye.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="@dimen/button_height"
```
[the remark this answers is not in the document]
Comment (nfillion21) [reply phoenix-android/src/main/res/drawable/ic_eye.xml]: Done in e4d8b460

File: phoenix-android/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -596,7 +596,7 @@
     <string name="about_privacy_link">Privacy</string>
     <string name="about_terms_link">Terms</string>
 
-    <!-- //////////////// settings: options for payments //////////////// -->
+  <!-- //////////////// settings: options for payments //////////////// -->
```
[the remark this answers is not in the document]
Comment (nfillion21) [reply phoenix-android/src/main/res/values/strings.xml]: Done in 3d53ceec

File: phoenix-android/src/main/res/drawable/ic_eye.xml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    android:height="@dimen/button_height"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
```
[the remark this answers is not in the document]
Comment (nfillion21) [reply phoenix-android/src/main/res/drawable/ic_eye.xml]: Done in 5e5a5bc5

File: phoenix-android/src/main/kotlin/fr/acinq/phoenix/android/settings/LogsView.kt [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+import androidx.core.content.FileProvider
+import fr.acinq.phoenix.android.R
+import fr.acinq.phoenix.android.components.Card
+import fr.acinq.phoenix.android.components.SettingButton
+import fr.acinq.phoenix.android.components.SettingHeader
+import fr.acinq.phoenix.android.components.SettingScreen
+import fr.acinq.phoenix.android.navController
+import fr.acinq.phoenix.android.utils.logger
+import fr.acinq.phoenix.legacy.BuildConfig
+import fr.acinq.phoenix.legacy.utils.Logging
```
[the remark this answers is not in the document]
Comment (nfillion21) [reply phoenix-android/src/main/kotlin/fr/acinq/phoenix/android/settings/LogsView.kt]: Done in 0b1e45fb

File: phoenix-android/src/main/kotlin/fr/acinq/phoenix/android/settings/LogsView.kt [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+@Composable
+fun LogsView() {
+    val log = logger("LogsView")
+    val nc = navController
+    val context = LocalContext.current
+
+    val logFile = Logging.getLastLogFile(LocalContext.current)
```
[the remark this answers is not in the document]
Comment (nfillion21) [reply phoenix-android/src/main/kotlin/fr/acinq/phoenix/android/settings/LogsView.kt]: Done in a76c83b6

**Review by dpad85 — APPROVED** [review]

**Pull request merged**